skeletons/classname.xml

```diff
@@ -1,105 +1,132 @@
 <?xml version="1.0" encoding="utf-8"?>
-<phpdoc:classref xml:id="class.classname"
- xmlns:phpdoc="http://php.net/ns/phpdoc"
- xmlns:xlink="http://www.w3.org/1999/xlink"
- xmlns:xi="http://www.w3.org/2001/XInclude"
- xmlns="http://docbook.org/ns/docbook">
+<reference xml:id="class.classname" role="class" xmlns="http://docbook.org/ns/docbook" xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:xi="http://www.w3.org/2001/XInclude">
  <title>The ClassName class</title>
  <titleabbrev>ClassName</titleabbrev>
  
  <partintro>
- 
-<!-- {{{ ClassName intro -->
+
+  <!-- {{{ ClassName intro -->
   <section xml:id="classname.intro">
    &reftitle.intro;
-   <para>
+   <simpara>
     Description of the class.
-   </para>
+   </simpara>
   </section>
-<!-- }}} -->
- 
+  <!-- }}} -->
+
   <section xml:id="classname.synopsis">
    &reftitle.classsynopsis;
- 
-<!-- {{{ Synopsis -->
-   <classsynopsis>
-    <ooclass><classname>ClassName</classname></ooclass>
- 
-<!-- {{{ Class synopsis -->
-    <classsynopsisinfo>
-     <ooclass>
-      <classname>ClassName</classname>
-     </ooclass>
- 
-<!-- If the class extends another one, use this
-     <ooclass>
-      <modifier>extends</modifier>
-      <classname>BaseClassName</classname>
-     </ooclass>
--->
- 
-<!-- If the class implements an interface, or two, use this (one oointerface element per interface)
-     <oointerface>
-      <interfacename>InterfaceName</interfacename>
-     </oointerface>
--->
-    </classsynopsisinfo>
-<!-- }}} -->
- 
-    <classsynopsisinfo role="comment">Properties</classsynopsisinfo>
-<!-- If the property is documented below (xml:id=classname.props) use this
+
+   <!-- {{{ Synopsis -->
+   <classsynopsis class="class">
+    <ooclass>
+     <classname>ClassName</classname>
+    </ooclass>
+
+    <!-- If the class extends another one, use this -->
+    <ooclass>
+     <modifier>extends</modifier>
+     <classname>ParentClass</classname>
+    </ooclass>
```

Comment:
Not 100% sure that uncommenting is a good idea, although that's a hint for the author to remove unnecessary sections.

Reply:
In general, I think we don't want commented-out boilerplate in skeletons, and there should be dummy text in anything that is expected to be filled out. We should add a QA script that trips on dummy text left in new files.